app/javascript/dashboard/routes/dashboard/settings/reports/components/FilterSelector.vue

```diff
@@ -1,6 +1,8 @@
 <template>
   <div class="flex flex-col md:flex-row justify-between mb-4">
-    <div class="md:grid flex flex-col filter-container gap-3 w-full">
+    <div
```

Comment:
In case you know what min-width you need for the columns, you can use this class to ensure that the columns arrange themselves nicely `grid-cols-[repeat(auto-fit,minmax(200px,1fr))]` 

Replace the 200px with the min width you want the item to be. Here's an example of how it works: https://play.tailwindcss.com/Qn9PheDb3D